docs/advanced/modules.md

```diff
@@ -0,0 +1,50 @@
+# Modules
+
+Modules are subapplications that are run as application plugins.
+They're used to define pieces of functionality that are pluggable to Decidim.
+
+Decidim's modules are no more than Ruby on Rails engines that should be required in the application's `Gemfile`.
+
+## Example
+
+A typical engine looks like the following:
+
+```ruby
+module Decidim
+  module Verifications
+    module Sms
```

Comment:
I think it is better to use a fictitious engine
Something like that should be better
```suggestion
    module MyVerification
```